README.md

```diff
@@ -73,20 +73,17 @@ $ npm install vorpal --save
 Create a `.js` file and add the following:
 
 ```js
-const vorpal = require('vorpal')();
-
-vorpal
-  .command('foo', 'Outputs "bar".')
-  .action(function(args, callback) {
-    this.log('bar');
-    callback();
-  });
-
-vorpal
-  .delimiter('myapp$')
-  .show();
+const vorpal = require("vorpal")();
+
+vorpal.command("foo", 'Outputs "bar".').action(function(args, callback) {

```

Comment:
Must say I liked the previous formatting (each call on a new line) a bit better

Reply:
Oops, looks like my prettier hit this one unintentionally.  Want me to break these back out?